api/api_test.go

```diff
@@ -405,8 +409,8 @@ func TestJsonWalletApi(t *testing.T) {
 	got, want = resp.StatusCode, http.StatusOK
 	assert.Equal(t, want, got)
 
-	// test get eligible layers / miner rewards
-	url = fmt.Sprintf("http://127.0.0.1:%d/v1/startmining", config.ConfigValues.JSONServerPort)
+	// test get statistics about init progress
+	url = fmt.Sprintf("http://127.0.0.1:%d/v1/stats", config.ConfigValues.JSONServerPort)
```

Comment:
we don't want to test startmining anymore ?

Reply:
I have a test for that before this one, that was a duplicate